ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoSuite.scala

```diff
@@ -443,6 +445,28 @@ private[dao] trait JdbcLedgerDaoSuite extends AkkaBeforeAndAfterAll with JdbcLed
       implicit ec: ExecutionContext): Future[(Offset, LedgerEntry.Transaction)] =
     store(divulgedContracts = Map.empty, offsetAndTx)
 
+  protected final def storeSync(commands: Vector[(Offset, LedgerEntry.Transaction)])(
+      implicit ec: ExecutionContext): Future[Vector[(Offset, LedgerEntry.Transaction)]] = {
+
+    import scalaz.Free
+    import scalaz.syntax.traverse._
+    import scalaz.std.vector._
+    import scalaz.std.scalaFuture._
+
+    val storeDelayed = (a: (Offset, LedgerEntry.Transaction)) => Free.liftFU(store(a))
```

Comment:
`liftFU` is strict; also, don't use `U` functions anymore.

```suggestion
    val storeDelayed = (a: (Offset, LedgerEntry.Transaction)) => Free.suspend(Free.liftF(store(a)))
```